src/shared/components/paginationControls/PaginationControls.tsx

```diff
@@ -44,6 +46,7 @@ export class PaginationControls extends React.Component<IPaginationControlsProps
         itemsPerPage: SHOW_ALL_PAGE_SIZE,
         itemsPerPageOptions: [10, 25, 50, 100],
         showAllOption: true,
+        textBeforeButtons:"",
         textBetweenButtons: "text btwn buttons",
```

Comment:
Actually, now we should name this as something like this: `textForShowMoreButton` or `textForMiddleButton`. Now, that button is functional too.